cmd/procs/description/descriptor_test.go

```diff
@@ -94,6 +94,16 @@ func (s *DescribeCmdTestSuite) TestDescribeCmdRunProcNotSupported() {
 	s.mockPrinter.AssertExpectations(s.T())
 }
 
+func (s *DescribeCmdTestSuite) TestDescribeCmdRunProcForUnauthorizedUser() {
+	s.mockProctorEngineClient.On("ListProcs").Return([]proc.Metadata{}, errors.New("Unauthorized")).Once()
```

Comment:
The test says for describe command, but we're mocking list proc here. Please fix the discrepancy.

Reply:
I wonder how the test is passing when we're calling describe command and setting expectation on list-procs?

Reply:
Both ProcList and Describe Job are calling JobList API. That's why we are only mocking ListProcs.